google/cloud/spanner/integration_tests/backup_integration_test.cc

```diff
@@ -36,116 +37,115 @@ namespace spanner {
   }
-  InstanceAdminClient instance_admin_client_;
-  DatabaseAdminClient database_admin_client_;
-  bool emulator_ = false;
-  std::string project_id_;
-  bool run_slow_backup_tests_ = false;
-};
 
-class BackupTestWithCleanup : public BackupTest {
  protected:
-  void SetUp() override {
-    BackupTest::SetUp();
-    instance_name_regex_ = std::regex(
-        R"(projects/.+/instances/(temporary-instance-(\d{4}-\d{2}-\d{2})-.+))");
-    instance_config_regex_ = std::regex(".*us-west.*");
-    if (!run_slow_backup_tests_) {
-      return;
-    }
-    auto s = spanner_testing::CleanupStaleInstances(project_id_,
-                                                    instance_name_regex_);
-    EXPECT_STATUS_OK(s) << s.message();
-  }
-  std::regex instance_name_regex_;
-  std::regex instance_config_regex_;
+  google::cloud::internal::DefaultPRNG generator_;
+  DatabaseAdminClient database_admin_client_;
 };
 
 /// @test Backup related integration tests.
-TEST_F(BackupTestWithCleanup, BackupTestSuite) {
-  if (!run_slow_backup_tests_ || emulator_) {
-    GTEST_SKIP();
-  }
-  auto generator = google::cloud::internal::MakeDefaultPRNG();
-  std::string instance_id =
-      google::cloud::spanner_testing::RandomInstanceName(generator);
-  std::string database_id =
-      google::cloud::spanner_testing::RandomDatabaseName(generator);
-  std::string restore_database_id =
-      google::cloud::spanner_testing::RandomDatabaseName(generator);
-  ASSERT_FALSE(project_id_.empty());
-  ASSERT_FALSE(instance_id.empty());
-  Instance in(project_id_, instance_id);
-
-  // Make sure we're using correct regex.
-  std::smatch m;
-  auto full_name = in.FullName();
-  EXPECT_TRUE(std::regex_match(full_name, m, instance_name_regex_));
-  std::string instance_config = spanner_testing::PickInstanceConfig(
-      project_id_, instance_config_regex_, generator);
-  ASSERT_FALSE(instance_config.empty()) << "could not get an instance config";
-  future<StatusOr<google::spanner::admin::instance::v1::Instance>> f =
-      instance_admin_client_.CreateInstance(
-          CreateInstanceRequestBuilder(in, instance_config)
-              .SetDisplayName("test-display-name")
-              .SetNodeCount(1)
-              .Build());
-  auto instance = f.get();
+TEST_F(BackupTest, BackupTest) {
+  if (!RunSlowBackupTests() || Emulator()) GTEST_SKIP();
 
-  EXPECT_STATUS_OK(instance);
+  InstanceAdminClient instance_admin_client(MakeInstanceAdminConnection(
+      ConnectionOptions{}, spanner_testing::TestRetryPolicy(),
+      spanner_testing::TestBackoffPolicy(),
+      spanner_testing::TestPollingPolicy()));
 
-  Database db(project_id_, instance_id, database_id);
-
-  auto database_future = database_admin_client_.CreateDatabase(db);
-  auto database = database_future.get();
+  Instance in(ProjectId(), spanner_testing::RandomInstanceName(generator_));
+  auto instance_config = spanner_testing::PickInstanceConfig(
+      in.project_id(), std::regex(".*us-west.*"), generator_);
```

Comment:
Maybe a comment to document why `us-west` is special?  I have some vague recollection that we have more room to create instances there or something.

Reply:
https://github.com/googleapis/google-cloud-cpp-spanner/pull/1403 appears to be the origin, but doesn't list a reason.